Makefile

```diff
@@ -88,23 +100,16 @@ check-pbfiles: pbfiles
 	  git diff --no-ext-diff --exit-code -- protos/*/*.pb.*
 	)
 
-e2etest-setup: tools
-	setup-envtest use
-
-## --------------------------------------
-## Generate
-## --------------------------------------
-
 .PHONY: generate
-generate: generate-go generate-manifests # Generate code, manifests etc.
+generate: generate-go generate-manifests ## Generate code, manifests etc.
```

Comment:
when i do a `make generate` and then a `make lint`, `make lint` fails. is that expected behavior?